source/includes/gcp/_firewall_rules.md

```diff
@@ -141,6 +141,50 @@ Retrieve a firewall rules in a given [network](#gcp-networks)
 `targetTags`<br/>*Array[string]* | A list of tags that controls which instances the firewall rule applies to. If targetTags are specified, then the firewall rule applies only to instances in the VPC network that have one of those tags. If no targetTags are specified, the firewall rule applies to all instances on the specified network.
 `action`<br/>*string* | The firewall rule type: 'allow' (allow traffic) or 'deny' (deny traffic)
 
+<!-------------------- CREATE A FIREWALL RULE -------------------->
+
+#### Create a firewall rule
+
+<code>CREATE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/firewallrule</code>
+
+
+```shell
+curl -X POST \
+  -H "MC-Api-Key: your_api_key" \
+  -d "request_body" \
+  "https://cloudmc_endpoint/v1/services/gcp/test-area/firewallrule"
+```
+```json
+{
+  "action": "allow",
+  "all": true,
+  "direction": "INGRESS",
+  "name": "fw-rd-tuv",
+  "priority": 1000,
+  "range": "0.0.0.0/0",
+}
+```
+
+
+Creates a new allow all firewall rule for the specified range.
```

Comment:
The doc should be more general, not just about the example above. This seems like we always create new allow all rules

Reply:
I'll add a specific rule in the listing / retrieving

Reply:
> The doc should be more general, not just about the example above. This seems like we always create new allow all rules

Shall we show how to create allow all, and specific ?

Reply:
Changed.